src/modules/WhatsNew/WhatsNew.module.css

```diff
@@ -0,0 +1,49 @@
+@import '_colors.css';
+@import '_fonts.css';
+
+.whats-new {
+  width: 25%;
+  padding: 0.93rem 0.3rem;
+  color: $color-text-secondary;
+  font-weight: 500;
+  font-size: $medium-font-size;
+
+  &__heading {
+    margin: 0;
+    padding: 0.3rem 0.3rem 0;
+    font-weight: 500;
+  }
+
+  &__list {
+    padding: 0.5rem;
+  }
+
+  &__item {
+    display: flex;
+    justify-content: space-between;
+    padding: 0.8rem 0;
+  }
+
+  &__link {
+    padding-left: 0.8rem;
+  }
+
+  &__number-circle {
```

Comment:
I think we can create common component for that Icon

I need the same icon in menu 
<img width="310" alt="image" src="https://user-images.githubusercontent.com/15625912/162441425-e7f7b03e-cb83-4670-b58e-cc3e315ec5c9.png">

Reply:
@VladSolodjankin The number circle?